ambari-server/src/main/resources/common-services/ATLAS/0.1.0.2.3/package/scripts/metadata.py

```diff
@@ -149,9 +151,18 @@ def metadata(type='server'):
                                        roles = [params.infra_solr_role_atlas, params.infra_solr_role_ranger_audit, params.infra_solr_role_dev],
                                        new_service_principals = [params.atlas_jaas_principal])
 
-      create_collection('vertex_index', 'atlas_configs', jaasFile)
-      create_collection('edge_index', 'atlas_configs', jaasFile)
-      create_collection('fulltext_index', 'atlas_configs', jaasFile)
+      create_collection('vertex_index', 'atlas_configs', jaasFile,
+            trust_store_password =  default('configurations/ranger-atlas-policymgr-ssl/xasecure.policymgr.clientssl.truststore.password', None),
```

Comment:
Why to use ranger-plugin related trustore properties ?

adding @vishalsuvagia for review here.

Reply:
The plugin will need that information anyways to contact Solr later in to populate the audit entries

Reply:
That ok, but need to use atlas-side property here. What if atlas is not using ranger-plugin.

Reply:
@amerissa , @fimugdha seems right, in order to create a collection which would be used by Atlas Server, truststore setup for Ranger Atlas plugin should not be passed / used. The collection can even be in use by Atlas Server when Ranger Atlas plugin will not be enabled.

Reply:
Makes sense. Atlas does have a truststore property truststore.file; however, it only uses jceks to store the password for it. Do you guys know a way in python to extract the password from JCEKS?